dataprofiler/labelers/char_load_tf_model.py

```diff
@@ -376,7 +386,7 @@ def fit(
             history[metric_label] = model_results[i]
 
         if val_data:
-            f1, f1_report = self._validate_training(val_data)
+            f1, f1_report = self._validate_training(val_data)  # type: ignore
```

Comment:
why the `ignore` here?

Reply:
Is there a way we can suggest to mypy that it can't be? is there where overload signatures come into play?
https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading

Reply:
I tried adding overloading, but it doesn't work because `pandas` does not have any type stubs right now, so `pd.DataFrame` evalutes to `Any`. I found this potential [solution](https://stackoverflow.com/questions/63214123/mypy-overload-function-with-numpy-ndarray-and-pandas-dataframe-signature-parame).